test/helper.sh

```diff
@@ -14,6 +27,8 @@ function load_image() {
     if ! docker image inspect "${image}" >/dev/null 2>&1; then
         docker pull "${image}"
     fi
+
+    cmd_must_exist "kind"
```

Comment:
Maybe it's better to add all dependency checks at the beginning of the test, rather than at every location.

Reply:
add 
# check dependency
cmd_must_exist "kind"
cmd_must_exist "docker"
cmd_must_exist "kubectl"
cmd_must_exist "helm"